### DISCUSSION TITLE
Does the AC S4-W12JARXA capable w/ this plugin?

### DISCUSSION THREAD

**Discussion by VCTGomes** [nVuln/homebridge-lg-thinq#183] [Q&A]
I'm looking for some AC capable with HomeKit, but in Brazil I don't have many options.

I'd like to know if this plugin could work properly with the LG AC S4-W12JARXA. I didn't buy it to test it.

**Comment by nVuln** [comment]
I think this model is working with this plugin, as advertising this model can be controlled via LG ThinQ mobile app, it's enough to work
